docs/source/config_file.rst

```diff
@@ -439,6 +439,11 @@ These options may only be set in the global section (``[mypy]``).
 ``show_absolute_path`` (bool, default False)
     Show absolute paths to files.
 
+``ignore_error_codes`` (comma-separated list of strings)
+    Ignore errors with these error codes in given files or directories.
+    See :option:`--ignore-error-codes <mypy --ignore-error-codes>` for
+    more details.
```

Comment:
Really tiny consistency nit: we use "for more information" about and "more details here". Maybe use "for more information" here as well.